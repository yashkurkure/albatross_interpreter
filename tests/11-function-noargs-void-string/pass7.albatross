var s0 string := "xyaNJuDY95uNuHub83" ;
var s1 string := "q1IQO8boTJvia9EA" ;
var s2 string := "cyE56GjDmX3g" ;
var s3 string := "V" ;
var s4 string := "TtebNb1qE" ;
var s5 string := "3sD2MgOJK" ;
var s6 string := "f4IXDmK0MenYVHtR" ;
var s7 string := "t915xEHpFgvk6RcipsZ" ;
var s8 string := "Yw2" ;
var s9 string := "0COSpI" ;
fun fun58047 string () { return "2bbVo8bs8pQ";}
fun fun24278 void () { printstring(s8);}
fun fun38757 string () { return s9;}
fun fun7110 void () { printstring(s0);}
fun fun48579 string () { return "xEq00NQV5fp";}
fun fun6746 void () { printstring("XgSB2DAus6D");}
fun fun49429 string () { return s2;}
fun fun62314 void () { printstring(s8);}
fun fun43306 string () { return s1;}
fun fun44994 void () { printstring("g4YJ0JiYss");}
fun fun9794 string () { return s5;}
fun fun34509 void () { printstring("bk");}
fun fun51849 string () { return "VWpdRh6lSoOq";}
fun fun3455 void () { printstring("cBOHYzN");}
fun fun20759 string () { return s0;}
fun fun41817 void () { printstring("63B");}
fun fun44755 string () { return s5;}
fun fun36371 void () { printstring(s2);}
fun fun49124 string () { return "VkCdUgmkE4ZB";}
fun fun6916 void () { printstring(s5);}
fun36371();
printstring(fun43306());
fun41817();
printstring(fun49429());
fun6746();
printstring(fun48579());
fun44994();
printstring(fun51849());
fun36371();
printstring(fun58047());
fun6746();
printstring(fun20759());
fun34509();
printstring(fun58047());
fun41817();
printstring(fun43306());
fun6916();
printstring(fun38757());
fun6916();
printstring(fun49124());
exit(0);
