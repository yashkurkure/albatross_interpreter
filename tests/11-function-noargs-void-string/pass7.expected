cyE56GjDmX3gq1IQO8boTJvia9EA63BcyE56GjDmX3gXgSB2DAus6DxEq00NQV5fpg4YJ0JiYssVWpdRh6lSoOqcyE56GjDmX3g2bbVo8bs8pQXgSB2DAus6DxyaNJuDY95uNuHub83bk2bbVo8bs8pQ63Bq1IQO8boTJvia9EA3sD2MgOJK0COSpI3sD2MgOJKVkCdUgmkE4ZB
